### ISSUE TITLE
Allow setting the language for the google places api

### ISSUE THREAD

**Issue by bogdanbalc** [Tintef/react-google-places-autocomplete#146]

**Comment by Tintef** [comment]
@bogdanbalc This is kinda doable right now. If you don't rely on using the `apiKey` prop and inject the script yourself you can configure it however you want.

Nevertheless, this could be a nice (and simple) addition

**Comment by bogdanbalc** [comment]
You have a point. I could just inject the script myself. In this scenario where you can customize the script however you want ... I believe it could be a nice api to give users the ability to inject the script they constructed with a utility method like we have for getting coordinates for example.
What do you think?

**Comment by Tintef** [comment]
Yeah, I was thinking of a new prop, lets say `apiOptions` and the library uses that prop to build the appropiate script tag.

Example:
```
<GooglePlacesAutocomplete
  apiKey="*****************"
  apiOptions={{ language: 'es-419' }}
/>
```

Would inject:
```
<script
  type="text/javascript"
  src="https://maps.googleapis.com/maps/api/js?key=${apiKey}&libraries=places&language=es-419"
/>
```

**Comment by bogdanbalc** [comment]
@Tintef sounds great. Can you find the time to implement something like that? If not I can try to create a PR for this...

**Comment by Tintef** [comment]
Probably not in the short term -- next two or so weeks -- 

I would love to review a PR :)

**Comment by bogdanbalc** [comment]
Here it is: [PR](https://github.com/Tintef/react-google-places-autocomplete/pull/148)

**Comment by Tintef** [comment]
[PR](https://github.com/Tintef/react-google-places-autocomplete/pull/148) merged and feature release on v3.2.0
